config_templates/sp_authorization.yaml

```
@@ -0,0 +1,15 @@
+module: satosa.micro_services.sp_authorization_microservice.SpAuthorization
+name: SpAuthorization
+
+config:
+  global_cfg_filepath: filepath
+  unauthorized_redirect_url: url
+  filter_config:
+    - check_group_membership_attr: check_group_membership_attr
+    - vo_short_names_attr: vo_short_names
+    - registrar_url: registrar_url
+    - allow_registration_attr: allow_registration_attr
+    - registration_link_attr: registration_link_attr
+    # TODO skip_notification_sps should be an array
+    - skip_notification_sps: skip_notification_sps
+    - handle_unsatisfied_membership: handle_unsatisfied_membership
```

```suggestion
  filter_config:
    check_group_membership_attr: check_group_membership_attr
    vo_short_names_attr: vo_short_names
    registrar_url: registrar_url
    allow_registration_attr: allow_registration_attr
    registration_link_attr: registration_link_attr
    skip_notification_sps:
      - sp1
      - sp2
      - sp3
    handle_unsatisfied_membership: handle_unsatisfied_membership
```
This is equivalent to:
```json
{
  "filter_config": {
    "check_group_membership_attr": "check_group_membership_attr",
    // ....
    "skip_notification_sps": ["sp1", "sp2", "sp3"]
  }
}
```